docs/api/assert/closeTo.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page-api
+title: assert.closeTo()
+excerpt: Check that an object contains certain properties.
+groups:
+  - assert
+redirect_from:
+  - "/assert/closeTo/"
+version_added: "unreleased"
+---
+
+`closeTo( actual, expected, delta, message = "" )`
+
+Check that an object contains certain properties.
+
+| name | description |
+|------|-------------|
+| `actual` (number) | Expression being tested |
+| `expected` (number) | Known comparison value |
+| `delta` (number) |  |
+| `message` (string) | Short description of the actual expression |
+
+
+TODO: close, close-enough, roughlyEquals, closeTo, approximate
```

Comment:
Did you mean to leave this in? I don't think it needs to be user-facing. And as a user, I don't know what this means. Are all of these methods coming? `closeTo` is part of the list. Is there more to do on that one?

Reply:
These are keywords I meant to incorporate in the long-form description to help with search and discovery. I thought I had finished the docs, but I must have dreamed that. Consider this markdown file incomplete :)

Reply:
Fixed.